README.md

```diff
@@ -40,7 +40,8 @@ tools installed: gcc, binutils, flex, bison, etc.
 ## Ubuntu 14.04
 ```
 $ sudo apt-get install make unrar autoconf automake libtool gcc g++ gperf \
-    flex bison texinfo gawk ncurses-dev libexpat-dev python sed
+    flex bison texinfo gawk ncurses-dev libexpat-dev python sed python_serial \
+    git
```

Comment:
What's the point of having git here? And did you actually test the changes you propose? (Hint: look for typos in your changes).